app/exawind/exawind.cpp

```diff
@@ -209,6 +209,28 @@ int main(int argc, char** argv)
                                  : false;
     sim.set_holemap_alg(holemap_alg);
 
+    const YAML::Node& composite_mesh = node["composite_mesh"];
+    const int num_composite = composite_mesh.size();
```

Comment:
you probably want to static_cast the size to `int` to be safe.

Reply:
I added a static_cast<int> here.